Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -195,7 +201,9 @@ private void CreateRange()
             else
             {
                 this.lower = limits[0];
-                this.upper = limits[1];
+                this.upper = limits[1] > Int32.MaxValue
+                    ? limits[1]
+                    : limits[1] + 1;

```

Comment:
The limits[1] + 1 will prevent the 255th number from being 255.  

Random.Next(1,255) will never return 255 because it excludes the upper limit.  My change will make the call Random.Next(1,256) so that 255 could be returned.  However, Random.Next takes (Int32,Int32) which means it will break if the upper bound is larger than Int32.MaxValue.  So I only do this if the limit[1] is less than Int32.MaxValue.  Now that I think about it, the check should be >= Int32.MaxValue. 

Hope this explains it better.